src/components/service-map-marker.js

```diff
@@ -1,28 +1,48 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { Marker, Popup } from 'react-leaflet';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
 
-const ServiceMapMarker = props => {
-  const { record } = props;
-  const { LATITUDE, LONGITUDE, PROVIDER_NAME, SERVICE_NAME, ORGANISATION_PURPOSE, PHYSICAL_ADDRESS } = record;
+export default class ServiceMapMarker extends Component {
+  static propTypes = {
+    FSD_ID: PropTypes.string,
+    LATITUDE: PropTypes.string,
+    LONGITUDE: PropTypes.string,
+    PROVIDER_NAME: PropTypes.string,
+    SERVICE_NAME: PropTypes.string,
+    ORGANISATION_PURPOSE: PropTypes.string,
+    PHYSICAL_ADDRESS: PropTypes.string
+  };
 
-  const position = { lat: Number(LATITUDE), lng: Number(LONGITUDE) };
+  render() {
+    const { record } = this.props;
+    const { FSD_ID, LATITUDE, LONGITUDE, PROVIDER_NAME, SERVICE_NAME, ORGANISATION_PURPOSE, PHYSICAL_ADDRESS} = record;
 
-  return (
-    <div>
-      {LATITUDE && LONGITUDE && (
-        <Marker position={position}>
-          <Popup>
-            <span>
-              <h5>{PROVIDER_NAME}</h5>
-              <h6>{SERVICE_NAME}</h6>
-              <p>{ORGANISATION_PURPOSE}</p>
-              <p>{PHYSICAL_ADDRESS}</p>
-            </span>
-          </Popup>
-        </Marker>
-      )}
-    </div>
-  );
-};
+    const position = { lat: Number(LATITUDE), lng: Number(LONGITUDE) };
 
-export default ServiceMapMarker;
+    return (
+      <div>
+        {LATITUDE && LONGITUDE && (
+          <Marker position={position}>
+            <Popup>
+              <span>
+              <h5>
```

Comment:
This `h5` looks a bit weird - it's smaller than the body text. I think we should go with `h4`

<details>
<summary>With H5</summary>

<img width="365" alt="Screen Shot 2019-09-19 at 1 07 07 PM" src="https://user-images.githubusercontent.com/1615322/65202397-701c2b80-dade-11e9-94e6-e62bf4a707d3.png">
</details>

<details>
<summary>With H4</summary>

<img width="374" alt="Screen Shot 2019-09-19 at 1 07 35 PM" src="https://user-images.githubusercontent.com/1615322/65202520-8a560980-dade-11e9-8b78-50ae053371e4.png">

</details>

```suggestion
              <h4>
```